app/src/main/kotlin/co/netguru/baby/monitor/client/feature/machinelearning/AacRecorder.kt

```diff
@@ -21,21 +21,25 @@ class AacRecorder {
 
     fun startRecording(): Completable = Completable.fromAction {
         Timber.i("starting recording")
-        bufferSize = AudioRecord.getMinBufferSize(SAMPLING_RATE, AudioFormat.CHANNEL_IN_MONO, AudioFormat.ENCODING_PCM_16BIT) * 2
+        bufferSize = AudioRecord.getMinBufferSize(
+            SAMPLING_RATE,
+            AudioFormat.CHANNEL_IN_MONO,
+            AudioFormat.ENCODING_PCM_16BIT
+        ) * 2
         if (bufferSize == AudioRecord.ERROR || bufferSize == AudioRecord.ERROR_BAD_VALUE) {
             bufferSize = SAMPLING_RATE * 2
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/f1e62ee67a21a21873bbc26c24839e1cf2930e4c/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/machinelearning/AacRecorder.kt#L30"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>